website/content/api-docs/secret/identity/entity.mdx

```diff
@@ -400,6 +400,11 @@ Note that if these entities contain aliases sharing the same mount accessor, the
 entities must be merged one at a time. This is because each entity can only have one alias with each mount accessor - for more
 information, see the [identity concepts page](/vault/docs/concepts/identity).
 
+<Warning>
+It is recommended to execute merges from the primary cluster's leader node. For additional information, please refer to the release notes of the respective version in use.
```

Comment:
```suggestion

We recommended limiting merging to the leader node of your primary cluster.

```
Style correction: avoid possessives. 

I'm not sure the second sentence is helpful since we don't actually link to anything. What problem is the warning meant to solve?